pym/bob/input.py

```diff
@@ -2329,6 +2332,13 @@ def __parseUserConfig(self, fileName, relativeIncludes=None):
         if not self._ignoreCmdConfig:
             self.__commandConfig = updateDicRecursive(self.__commandConfig, cfg.get("command", {}))
         self.__buildHooks.update(cfg.get("hooks", {}))
+        # change color mode
+        ui = cfg.get('ui', {})
+        if ui:
+            mode = ui.get('color', None)
+            if mode:
+                # note: this is applied after the cmd line options are parsed
+                setColorMode(mode)
```

Comment:
You can use `updateDicRecursive()` to merge the dicts (see above). Also, the order is not as intended as the command line option should always overrule whatever is in the config files.